Docs/Book/C++Examples/example11.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Working with 3D molecules - example11.cpp
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+#include <Geometry/point.h>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/MolOps.h>
+#include <GraphMol/MolAlign/AlignMolecules.h>
+#include <GraphMol/DistGeomHelpers/Embedder.h>
+#include <GraphMol/ForceFieldHelpers/MMFF/MMFF.h>
+#include <GraphMol/ForceFieldHelpers/UFF/UFF.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+
+int main( int main , char **argv ) {
+
+  RDKit::ROMOL_SPTR mol( RDKit::SmilesToMol( "C1CCC1OC" ) );
+  RDKit::ROMOL_SPTR mol1( RDKit::MolOps::addHs( *mol ) );
+  // Original distance geometry embedding
+  RDKit::DGeomHelpers::EmbedMolecule( *mol1 , 0 , 1234 );
+  RDKit::UFF::UFFOptimizeMolecule( *mol1 );
+  
+  // new Riniker and Landrum CSD-based method
```

Comment:
ah, yes, you use it once; I see it now.
I don't see any value in having the full set of parameters show up in a tutorial document. They are difficult to follow and confusing (and, yes, I know that I'm responsible for this API...). The parameter class is much cleaner.